packages/typespec-vscode/src/extension.ts

```diff
@@ -30,346 +24,37 @@ export async function activate(context: ExtensionContext) {
   );
 
   context.subscriptions.push(
-    commands.registerCommand("typespec.restartServer", restartTypeSpecServer),
+    commands.registerCommand("typespec.restartServer", async () => {
+      if (client) {
+        await client.restart();
+      }
+    }),
+  );
+
+  context.subscriptions.push(
+    vscode.workspace.onDidChangeConfiguration(async (e: vscode.ConfigurationChangeEvent) => {
```

Comment:
restart server when config changes.

Reply:
i see thats what I was looking for thanks!